runners/katacoda/index.ts

```diff
@@ -209,6 +209,18 @@ export class Katacoda extends Runner {
 
     }
 
+    runDockerCompose(step: Step, command: Command) : RunResult {
```

Comment:
miss the assertion accordingly checking that docker compose completes.

@GuentherJulian is there also a ways similar to runJavaServer to wait until the success has been reported? Otherwise, you could not step ahead and access the potential URL started by docker compose.

Reply:
@maybeec I think you refer to the console runner? I think we can do this the way as implemented in runServerJava for the console runner. So better wait until this pr is merged to test it.